src/transformers/models/vit_hybrid/configuration_vit_hybrid.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" ViT Hybrid model configuration"""
+
+import copy
+from typing import Dict
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+from ..auto.configuration_auto import CONFIG_MAPPING
+from ..bit import BitConfig
+
+
+logger = logging.get_logger(__name__)
+
+VIT_HYBRID_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "google/vit-base-r50-s16-384": "https://huggingface.co/vit-base-r50-s16-384/resolve/main/config.json",
```

Comment:
Same here regarding checkpoint naming. Would be cool to have hybrid in the name to avoid confusion with regular vit.

Reply:
Fixed in [93669f7](https://github.com/huggingface/transformers/pull/20550/commits/93669f7fb53bbdc56a73eb8b4c9cae553ad8cf94)